heartbeat/monitors/active/http/check.go

```diff
@@ -132,14 +134,100 @@ func checkHeaders(headers map[string]string) respValidator {
 	}
 }
 
-func checkBody(matcher []match.Matcher) bodyValidator {
-	return func(r *http.Response, body string) error {
-		for _, m := range matcher {
-			if m.MatchString(body) {
+func parseBody(b interface{}) ([]match.Matcher, []match.Matcher) {
+	var (
+		pm, nm []match.Matcher
```

Comment:
It'd be cleaner to just name the return values and avoid the extra var declaration here. I also wouldn't mind using longer var names for clarity. 

`func parseBody(b interface{}) (positiveMatch []match.Matcher, negativeMatch []match.Matcher) {`

Reply:
That makes sense.